### DISCUSSION TITLE
Using a project as "base map" for standalone datasets...doesn't work

### DISCUSSION THREAD

**Discussion by timbo-barnes** [opengisch/QField#4311] [Q&A]
I have a 'Basemap' project containing an online WMTS layer and an 'offline' MBtiles file with aerial photos.
I have a series of other projects which all should use this WMTS and MBtiles data as their basemap (to avoid including the large mbtiles file in multiple projects)

The Basemap project opens and displays the 'online' and 'offline' data ok (apart from the usual warnings that Qfield can't find the 'online' data if I disconnect my device from a data connection)

I then long press and set Basemap project  as 'individual datasets base map' to use it as a basemap in other projects.

I then open one of my other projects and expect that the imagery layers from the Basemap project will show? They do not show.
Is that how it's supposed to work or am I misinterpreting the help here?
Has anyone got this working successfully?

**Comment by GlaDal** [comment]
hi, I don't manage to find any documentation about this option. Do you know if there is ?

**Reply by timbo-barnes** [reply]
It's outlined here, but the information is quite sparse: https://docs.qfield.org/how-to/standalone-datasets/

**Comment by nirvn** [comment]
@timbo-barnes , this basemap option only works when you are opening say a gpkg or a shapefile; your description sounds like you are trying to open other projects and expect this one to show up (which it wouldn't).

**Reply by timbo-barnes** [reply]
Thanks @nirvn 
Ok, I guess I misunderstood for what that option was for.
I assumed it was a way to allow re-use of large datasets (i.e. imagery/rasters) between projects.
In our use case (and presumably others), our field work is in geographically dispersed areas where there is no cell/wifi/data coverage so streaming online WMS/WMTS imagery as a basemap for the vectors isn't feasible or reliable. 
I've managed to combine the aerial imagery for these areas into a single MBtiles file (a geopackage containing this many separate rasters caused hangs/crashes in Qfield), but these areas will be used in many different projects so, it makes sense to upload >store >download/sync _once_ to the device then _use_ many times.

Placing that raster file within a Qfield cloud project nicely handles the distribution of it. Are there plans to allow this 'basemap' functionality with rasters?
